packages/build/test-d/options/netlify_plugin_functions_util.ts

```diff
@@ -0,0 +1,35 @@
+import { OnPreBuild, NetlifyPluginUtils, ListedFunction, ListedFunctionFile } from '@netlify/build'
+import { expectType } from 'tsd'
+
+const testUtilsFunctionsList: OnPreBuild = function ({
+  utils: {
+    functions: { list },
+  },
+}: {
+  utils: NetlifyPluginUtils
+}) {
+  expectType<Promise<ListedFunction[]>>(list())
+  list()
```

Comment:
The additional second statement might not be needed, because `list()` is already being called in the above statement, so `tsc` and `tsd` are already checking its type.

The same applies to `listAll()` and `add('functionName')` below.